pkg/printers/internalversion/printers_test.go

```diff
@@ -1608,6 +1608,131 @@ func TestPrintPod(t *testing.T) {
 			},
 			[]metav1.TableRow{{Cells: []interface{}{"test20", "0/1", "Terminating", "0", "<unknown>"}}},
 		},
+		{
```

Comment:
I think worth adding a test case as in https://github.com/kubernetes/kubernetes/issues/107713:
init container succeeded, while the main container failed.